### PR TITLE
New package: trurl-0.8

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/trurl/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'trurl'
+pkgname=trurl
+version=0.2
+revision=1
+build_style=gnu-makefile
```
Comment (icp1994) [inline srcpkgs/trurl/template:5]:
```suggestion
build_style=gnu-makefile
make_check_target="test"
```

File: srcpkgs/trurl/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+makedepends="libcurl-devel"
```
Comment (icp1994) [inline srcpkgs/trurl/template:6]:
```suggestion
makedepends="libcurl-devel"
checkdepends="perl"
```

**Review by icp1994 — CHANGES_REQUESTED** [review]

File: srcpkgs/trurl/template (hunk 2 above)
Comment (TinfoilSubmarine) [reply srcpkgs/trurl/template:7]: Thanks!

**Review by icp1994 — APPROVED** [review]

**Comment by icp1994** [comment]
Any reason to push v0.6 while the newest tag is v0.8?

**Pull request merged**